ui/src/components/Pages/TrafficPage/TrafficPage.tsx

```diff
@@ -116,7 +116,11 @@ export const TrafficPage: React.FC<TrafficPageProps> = ({setAnalyzeStatus}) => {
             setLeftOffTop(null);
             setNoMoreDataTop(false);
         }
-        ws.current = new WebSocket(MizuWebsocketURL);
+        let websocketUrl = MizuWebsocketURL;
```

Comment:
I think we can extract this code from the component cuz we use only the "api" var and MizuWebsocketURL, and they are both not related to the component